.prettierrc.json

```diff
@@ -0,0 +1,14 @@
+{
+	"$schema": "http://json.schemastore.org/prettierrc",
+	"trailingComma": "all",
```

Comment:
It looks like this rule was only applied to a few `.ts` files. Any idea why?

Reply:
Because I want to spread applying this across several PRs, starting off with my type definitions.